src/templates/calendar/calendar.test.js

```diff
@@ -0,0 +1,16 @@
+describe('Make sure calendar loads', () => {
+  it('should load component', () => {
+    cy.visit('http://localhost:3000');
+    cy.get('select').select('calendar1');
+    cy.get('.template-calendar').should('have.css', 'background-color', 'rgb(35, 85, 135)');
+  });
+
+  it('should have title', () => {
+    cy.visit('http://localhost:3000');
+    cy.get('select').select('calendar1');
```

Comment:
It's better to use a class or ID as selector, since other 'select' elements might exist.